assets/xml/objects/object_boss03.xml

```diff
@@ -1,93 +1,135 @@
 ﻿<Root>
+    <!-- 
+        Gyorg, his title card, the small fishes he summons, the seaweed in his boss arena,
+        and the bubbles in his intro cutscene. Also contains an unused copy of Majora's Wrath's
+        whip assets, along with an unused duplicate of the small fishes.
+    -->
     <File Name="object_boss03" Segment="6">
```

Comment:
Nice work with the object.
There's a detail tho. Lately we are trying to preserve the order of the offsets of each extracted asset.
So, could you move those back to the original order this file had?

Reply:
Done, but can you please point to where this discussion happened? This is literally the first I've heard of it. I went and updated all my other asset PRs to preserve the order of the original assets, but some of those PRs were already approved, giving me the impression that this practice was supported.

Reply:
I couldn't find where the discussion happened (I think it happened somewhere in OoT). 
The main reason for this is objects having file splits themselves (those can be seen by the `possiblePadding` ZAPD generates). There's the intention to split those files, and ZAPD is currently able to do that, but we haven't decided yet if we want to do that.

If you are curious of how an splitted object file looks like, you could take a look at the object_human PR in OoT: https://github.com/zeldaret/oot/pull/1004/files

Reply:
Ahh, I see. I've been trying to follow MM conversations as much as possible, so I thought it strange that I had somehow missed something; turns out, it was an OoT discussion. And yeah, that makes total sense. Thanks for clarifying!